Classes_Homework_5/modules/dbconnect.py

```diff
@@ -0,0 +1,38 @@
+import sqlite3 as sql
+
+from exec_utils.configloader import Config
+from modules.file import Files
+
+cnf = Config()
+f = Files()
+
+label1, label2, label3 = cnf.get_values("LABELS", "news_label"), \
+                         cnf.get_values("LABELS", "ad_label"), \
+                         cnf.get_values("LABELS", "recipe_label")
+
+
+class DBconnection:
+    def __init__(self, db_name):
+        with sql.connect(f.get_path(db_name)) as self.con:
+            self.curs = self.con.cursor()
+
+    def go(self, statement, predicates=''):
+        """Execute sql statement"""
+        self.curs.execute(statement, predicates)
```

Comment:
Just for your info: besides the **execute** command,  we can use **executemany**, which will be more efficient for batch loading, and the performance difference will typically be quite large as your number of records gets large
https://www.kite.com/python/docs/sqlite3.Cursor.executemany